core/amber/src/main/python/core/models/table.py

```diff
@@ -1,10 +1,100 @@
-from typing import TypeVar
+from typing import Iterator, TypeVar, List
 
+from pampy import match
 import pandas
 
-TableLike = TypeVar("TableLike", pandas.DataFrame, pandas.DataFrame)
+from core.models import Tuple, TupleLike
+
+
+# @typing_extensions.runtime_checkable
```

Comment:
Any reason why this commented code is kept?

Reply:
removed